adapters/YahooAdapter/Public/Headers/VerizonMediaAdapter.h

```diff
@@ -16,4 +16,3 @@
 #import <UIKit/UIKit.h>
 
 // Import any public headers below.
-#import <VerizonMediaAdapter/GADMVerizonPrivacy.h>
```

Comment:
This file has the old name, and, is effectively n/a since the privacy files have been removed, probably this file and the module map can be deleted as well; noting that I have yet to verify the adapter for the updated Yahoo SDK privacy APIs fully.

Reply:
Removed from the modulemap.